GitBranches/Program.cs

```diff
@@ -13,7 +13,7 @@ static void Main(string[] args)
             Console.WriteLine("Demo Git");
             Console.WriteLine("Estas son las preguntas que vamos a contestar: ");
             Console.WriteLine("1. Cunato timepo llevas en Hartmann?");
-            Console.WriteLine("Esta es la rama de Robert");
+            Console.WriteLine("Esta es la rama de Enrico");
```

Comment:
The idea is to have answers of everyone.
Please fix this.
Plus you've been longer than Robert.